pmlc/runtime/symbol_registry.cc

```diff
@@ -0,0 +1,33 @@
+// Copyright 2020 Intel Corporation
```

Comment:
This doesn't seem to be runtime-specific, can we leave it on the compiler?

Reply:
I'm not sure when the compiler would ever use this...?  Seems like the symbols are used at JIT time; I was thinking of that as a post-compilation step.